### ISSUE TITLE
Research what information and/or metrics would help students identify what they want to do or who they want to work for

### ISSUE THREAD

**Issue by aflies1** [promazo/Gen2-Consulting#7]
Create a list of information that would help a student determine what job and what company to work for (ie. metrics, geography, culture..) Rank order the list on what you think would be most important and why. Determine how the top information would be best consumed from a technical standpoint (How would the student want it displayed?) Submit list to the ProMazo manager (Adam) for review.

**Comment by ninashen-09** [comment]
https://docs.google.com/forms/d/1MRFy2vBnUl_oXfzdOtfKgvcTwFnRpleCDffumc5rvJA/edit

**Comment by aflies1** [comment]
Comments on Survey:

- I like how short it is, should get a lot of responses

- Why just choose "top 3"?  Would it be more valuable to us to do a forced rank? That way we'd be able to identify what the bottom 3 are as well. (Questions 3 and 4)

- Change question 5 to "What metrics about companies would help you make a more informed decision on who to apply/ work for? (Top 3 is good here, but again would it be better to do a forced rank?)

- Question 6 - change phrasing of the question to assume that the student taking the survey is still in their career search. (same as throughout the survey)

- Change question 8 to something along the lines of "what is something that has helped you in your career search that could help another student?'

- Change the final question to reflect that they may still be in their search.

Overall this looks great - change the phrasing to reflect that the majority of students taking this will probably still be in their career/job/internship search. I would also suggest making a forced rank so we can get a good view on how students view all the options (maybe add a question after each of "what not listed if anything would fall in your top 3?"

**Comment by irynakmyts** [comment]
Dear Adam, 

Thank you for your valuable comments. 

I agree that we would miss some important information, if we only ask for top 3. Therefore, I changed the format of Q3 and Q4 to a rank. I also added question after Q3, and changed the phrasing of Q5, Q6, Q8, Q9, as you suggested.

The survey is now ready to be send (use this link https://forms.gle/ywE7GQg98PoaRehA9 ) , and I emailed the team asking to spread it out. 

All the best,
Iryna
